gen/CDP/Gen/Program.hs

```diff
@@ -24,18 +24,11 @@ import qualified CDP.Gen.Snippets as Snippets
 
 ----------- Constants from the handwritten runtime of the CDP library ----------- 
 
-eventClassName, eventResponseName, handleTypeName, commandClassName, sendCommandName, sendCommandResultName, emptyReturnSig :: T.Text
+eventClassName, eventResponseName, commandClassName, emptyReturnSig :: T.Text
```

Comment:
I think `commandClassName` is the only variable being used.
It would probably be better if it is inlined, since it is only used once.